js/src/annotations/osd-region-draw-tool.js

```diff
@@ -122,25 +134,31 @@
       var _this = this;
       _this.annotationsToShapesMap = {};
       var deferreds = jQuery.map(this.list, function(annotation) {
-        var deferred = jQuery.Deferred();
-        var shapeArray;
-        if (typeof annotation.on === 'object') {
-          if (annotation.on.selector.value.indexOf('<svg') != -1) {
-            shapeArray = _this.svgOverlay.parseSVG(annotation.on.selector.value, annotation);
+
+        var deferred;
+
```

Comment:
We create the deferred only if this is not a metadata annotation. See function above.